test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SearchTestsBase.cs

```diff
@@ -27,33 +30,83 @@ protected SearchTestsBase(TFixture fixture)
 
         protected async Task<Bundle> ExecuteAndValidateBundle(string searchUrl, params Resource[] expectedResources)
         {
-            return await ExecuteAndValidateBundle(searchUrl, searchUrl, expectedResources);
+            return await ExecuteAndValidateBundle(searchUrl, searchUrl, true, expectedResources);
+        }
+
+        protected async Task<Bundle> ExecuteAndValidateBundle(string searchUrl, bool sort, params Resource[] expectedResources)
+        {
+            return await ExecuteAndValidateBundle(searchUrl, searchUrl, sort, expectedResources);
         }
 
         protected async Task<Bundle> ExecuteAndValidateBundle(string searchUrl, string selfLink, params Resource[] expectedResources)
         {
             Bundle bundle = await Client.SearchAsync(searchUrl);
 
-            ValidateBundle(bundle, selfLink, expectedResources);
+            ValidateBundle(bundle, selfLink, true, expectedResources);
 
             return bundle;
         }
 
+        protected async Task<Bundle> ExecuteAndValidateBundle(string searchUrl, string selfLink, bool sort, params Resource[] expectedResources)
+        {
+            FhirResponse<Bundle> firstBundle = await Client.SearchAsync(searchUrl);
+
+            var firstExpectedResources = expectedResources.Length > 10 ? expectedResources.ToList().GetRange(0, 10).ToArray() : expectedResources;
```

Comment:
Can we have `10` as a variable that is also used to set the `_count` in search, just so this paging seems more obvious

Reply:
Alternatively, maybe pull this out into a pageSize field